addons/stock_calendar/models/procurement.py

```diff
@@ -303,27 +289,22 @@ def _procure_orderpoint_confirm(self, cr, uid, use_new_cursor=False, company_id=
                                     continue
 
                                 qty -= subtract_qty[op.id]
-
                                 qty_rounded = float_round(qty, precision_rounding=op.product_uom.rounding)
                                 if qty_rounded > 0:
-                                    proc_id = procurement_obj.create(cr, uid,
-                                                             self._prepare_orderpoint_procurement(cr, uid, op, qty_rounded, date=ndelivery, purchase_date=npurchase, group=group, context=context),
-                                                                     context=context)
-                                    tot_procs.append(proc_id)
-                                    orderpoint_obj.write(cr, uid, [op.id], {'last_execution_date': datetime.utcnow().strftime(DEFAULT_SERVER_DATETIME_FORMAT)}, context=context)
+                                    newprocurement = Procurementorder.create(self._prepare_orderpoint_procurement(op, qty_rounded, date=ndelivery, purchase_date=npurchase, group=group))
+                                    tot_procs += newprocurement
+                                    op.write({'last_execution_date': datetime.utcnow()})
                                 if use_new_cursor:
                                     cr.commit()
                         except OperationalError:
                             if use_new_cursor:
-                                orderpoint_ids.append(op.id)
+                                orderpoints += [op.id]
```

Comment:
when list is collection of ids then variable name should be orderpoint_ids, Don't be increase diff. :o: